src/index.js

```diff
@@ -7,8 +7,9 @@ import Parser from './parser';
 import Promise from 'bluebird';
 import Requester from './requester';
 import _ from 'lodash';
+import debugnyan from 'debugnyan';
 import methods from './methods';
-import request from 'request';
+import requestLogger from './logging/request-logger';
```

Comment:
What about using moving Requester inside ./request and export Requester and request already modified by the logger there?